modules/@apostrophecms/schema/ui/apos/components/AposInputRange.vue

```diff
@@ -108,7 +155,7 @@ export default {
   // adapted from http://danielstern.ca/range.css/#/
   .apos-range__input {
     width: 100%;
-    margin: 5px 0;
+    margin: 5px 0; // Question: Shoud we keep that margin? It creates more space than for other fields.
```

Comment:
@stuartromanek 
Question: Shoud we keep this margin? It creates more space than for other fields.